components/HomeScreen/Title/Title.utils.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @callback setGlobeOrientation
+ * @param {number}
+ */
+
+/**
+ * yieldRotateGlobe.
+ *
+ * @param {number} globeOrientation
+ * @param {function} setGlobeOrientation
+ */
```

Comment:
Your jsdoc is missing a return type. This is especially important since what you are returning is a callback being used elsewhere.